docs/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Is this UserStory done?
+
+You might be confused about what it really means for a User Story to be 'done'.
+
+Well you're in luck! We at the **DoD - Department of Done** have comprised a helpful checklist to see if your code meets our one true definition of done.
+
+The User Story is done if **and only if**:
+
+- [ ] User Story
+- - [ ] All Acceptance Criteria met
+- [ ] Code
+- - [ ] No more code needed
+- - [ ] No known bugs
+- [ ] Clean Code
+- - [ ] Comments have been created for new code
+- - [ ] ESLinter found no issues
```

Comment:
API uses dotnet and doesn't have eslint configured. How are we dealing with checkboxes that are only for editor/executor? or only for the api?

Reply:
I think one could leave these checkboxes where they are and just add editor/executor in brackets.